types/ramda/test/filter-tests.ts

```diff
@@ -62,6 +62,7 @@ import { Dictionary } from 'ramda/tools';
   let stringArray: string[];
 
   numberArray = R.filter(R.is(Number), unknownArray);
+  // $ExpectError
```

Comment:
Why error? Lines 64 and 66 are doing same things

Reply:
tslint says unknown[] can not be assigned to numbet[], I will look up what's happened.